Description: Make sure that the hardening flags are not removed.
Author: Benjamin Drung <bdrung@debian.org>

--- a/wscript
+++ b/wscript
@@ -295,9 +295,6 @@
         conf.msg("uncommited changed", changed and "yes" or "no")
         conf.env.VERSION = "%s (git commit: %s%s)" % (BASEVERSION, nam, dirty)
 
-    conf.env.append_unique('CFLAGS', ['-g', '-O0'])
-    conf.env.append_unique('CXXFLAGS', ['-g', '-O0'])
-
     if conf.options.with_profiling:
         conf.env.with_profiling = True
         conf.env.append_unique('CFLAGS', ['--coverage'])
